lib/js/jsconsole.nim

```diff
@@ -25,45 +25,69 @@ proc debugImpl(console: Console) {.importcpp: "debug", varargs.}
 proc infoImpl(console: Console) {.importcpp: "info", varargs.}
 proc errorImpl(console: Console) {.importcpp: "error", varargs.}
 proc warnImpl(console: Console) {.importcpp: "warn", varargs.}
+proc traceImpl(console: Console) {.importcpp: "trace", varargs.}
 
 proc makeConsoleCall(console: NimNode, procName: NimNode, args: NimNode): NimNode =
   result = newCall(procName, console)
   for c in args: result.add(c)
 
 macro log*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/log
   makeConsoleCall(console, bindSym "logImpl", args)
 
 macro debug*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/debug
   makeConsoleCall(console, bindSym "debugImpl", args)
 
 macro info*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
```

Comment:
i don't understand why all those macros are needed. this works and is much simpler:

```nim
proc error*(console: Console) {.importcpp, varargs.}
proc warn*(console: Console) {.importcpp, varargs.}
template exception*(console: Console, args: varargs[untyped]): untyped =
  error(console, args) # can't use importcpp here
# etc...
```
if you don't want to change this in this PR, I can do a followup PR to do that